core/jazz_metrics/components/utils.js

```diff
@@ -78,21 +86,26 @@ function getNameSpaceAndMetricDimensons(nameSpaceFrmAsset) {
       'aws/lambda': 'AWS/Lambda',
       'aws/cloudfront': 'AWS/CloudFront',
       'aws/s3': 'AWS/S3'
-    }
+    };
 
     if (Object.keys(nameSpaceList).indexOf(awsAddedNameSpace) > -1) {
       awsNameSpace = nameSpaceList[awsAddedNameSpace];
     } else {
       output_obj["isError"] = true;
-      output_obj["awsNameSpace"] = "Invalid";
+      output_obj["nameSpace"] = "Invalid";
```

Comment:
Can we use 'message' key when isError = true?

Reply:
updated